compiler/test/test-libs/provideStar.gr

```diff
@@ -0,0 +1,7 @@
+module ProvideStar
```

Comment:
Why is this named `ProvideStar` and then doesn't provide star (which I think has been removed)?